docs/credits.md

```diff
@@ -22,6 +22,7 @@ https://www.pucminas.br/destaques/Paginas/default.aspx), Brazil
 * David H. Hutchens - [Millersville University](https://www.millersville.edu/), Pennsylvania, USA
 * Tom Niget - [LEAT](https://leat.univ-cotedazur.fr/), [Polytech Nice-Sophia](https://polytech.univ-cotedazur.fr/),
 France
+* Marcin Orlowski, USA 
```

Comment:
Can I have it in that form please:

[Marcin Orlowski](http://MarcinOrlowski.com), Poland

Reply:
Oops, sorry, saw Palo Alto on your profile and assumed USA would be the right one. Fixing it right now